stack.yml

```diff
@@ -188,3 +188,10 @@ functions:
       write_debug: true
       read_debug: true
 
+  expose-metrics:
+    lang: go
+    handler: ./expose-metrics
+    image: functions/expose-metrics:0.1.0
+    environment:
+      prometheus_host: prometheus
```

Comment:
This won’t work on Kubernetes (do you know why?) and should be placed in an external env file for consistency.

Reply:
@alexellis for kubernetes we need to specify namespace so the host would be `prometheus.openfaas` I think.

Reply:
@alexellis do you think that period of time for metrics (`60m` by default) should be configurable via env var as @stefanprodan suggested? If so, do you think that putting that env var with prometheus host/port is a good idea? Like this:
```
environment:
  prometheus_host: prometheus.openfaas
  prometheus_port: 9090
  metrics_time_period: 60m
```

Reply:
Place it in `gateway_config.yml` please